Example/CoreDiagnostics/Tests/FIRCoreDiagnosticsTest.m

```diff
@@ -77,6 +89,7 @@ extern void FIRPopulateProtoWithInfoPlistValues(
 
 // FIRCoreDiagnosticsInterop.
 + (void)sendDiagnosticsData:(nonnull id<FIRCoreDiagnosticsData>)diagnosticsData;
+- (void)sendDiagnosticsData:(nonnull id<FIRCoreDiagnosticsData>)diagnosticsData;
```

Comment:
This method comes from FIRCoreDiagnosticsInterop, that also needs to be changed.

Reply:
Excuse me, not sure I got it... What needs to be changed?

Reply:
Sure! +sendDiagnosticsData is a method declared in `Interop/CoreDiagnostics/Public/FIRCoreDiagnosticsInterop.h`. You need to change the method declaration in that interop header (and how's it's called from FIRApp.m), or revert this change and change the implementation to use the +sharedInstance method. The latter would probably be easier.

Reply:
Right now the instance method is private and is used by the public static method, so everything will be working as before. Would you like to get rid of the static method and use `sharedInstance` instead?

Reply:
Got it--the diff can be hard to read with all the +s and -s. This part LGTM.